app/javascript/components/widgets/numbers/carousel.jsx

```diff
@@ -5,7 +5,7 @@ import gql from 'graphql-tag';
 import numeral from 'numeral';
 import { over, lensPath, inc } from 'ramda';
 import pluralize from 'pluralize';
-import { getMostRecentDay } from '../../../lib/datetime';
+import { getStartOfWeek } from '../../../lib/datetime';
```

Comment:
Love this commit's atomicity. However, we're importing a function here (`getStartOfWeek`) that doesn't yet exist. If we ran `git bisect`, we'd run into problems at this commit :( It's really important to ensure that each commit is a buildable one. I suggest: 

* Modifying `refactor: implement fns with date-fns and test ` to not (yet) remove `getMostRecentDay`, and remove the definition of `getStartOfWeek` from that commit
* reordering this commit to be the final commit, and move the code where you define the `getStartOfWeek` function into this commit

Make sense?